src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1623,7 +1623,10 @@ void AppHost::_PropertyChangedHandler(const winrt::Windows::Foundation::IInspect
 {
     if (e.PropertyName() == L"TitlebarBrush")
     {
-        auto nonClientWindow{ static_cast<NonClientIslandWindow*>(_window.get()) };
-        nonClientWindow->SetTitlebarBackground(_logic.TitlebarBrush());
+        if (_useNonClientArea)
+        {
+            auto nonClientWindow{ static_cast<NonClientIslandWindow*>(_window.get()) };

```

Comment:
Now here's a silly question . . . would it be better to implement `SetTitlebarBackground` as a virtual on IslandWindow to enable future customizability? Or, would that cause us to erroneously change the titlebar background color when the user didn't want it to change?

Reply:
it's not `titlebar.background` for a reason 😉 I dunno if there's a way to set the titlebar part of the frame color separate from the rest of the borders in the frame, and it's not a can of worms I want to investigate RN.